src/main/resources/dashboard/repo.hbs

```diff
@@ -71,6 +69,55 @@ only be able to download them. Nobody else will have any access to the repo.</p>
 <a href="https://maven.apache.org/plugins/maven-deploy-plugin/usage.html"><code>mvn deploy</code></a>
 and you download with
 <a href="https://maven.apache.org/plugins/maven-compiler-plugin/index.html"><code>mvn compile</code></a>.
+{{/eq}}{{#eq type "maven-proxy"}}
+<p>It's maven proxy repository, it proxies all request to remote Maven repository apply authentication
+and save artifacts in Artipie storage cache.</p>
```

Comment:
@g4s8 I'd suggest to rephrase:
Maven proxy repository proxies all request to remote Maven repository appling authentication
and saves artifacts in Artipie storage cache.